packages/mongo-migrate/src/generators/init/files/migration.config.ts__template__

```diff
@@ -1,8 +1,8 @@
 import { MigrationConfigFactory } from 'nx-mongo-migrate';
 
-const config: MigrationConfigFactory = () => ({
+const config: MigrationConfigFactory = async(factory = () => ({
```

Comment:
out of curiosity, why does this need to be wrapped in another function to return a function that returns the config? AFAIK, you don't need to wrap again to make this async as the executor awaits the result of whatever the default export of migration config. awaiting in JS also automatically wraps whatever you await in a promise, even it is not a promise.